src/models/client.ts

```diff
@@ -211,6 +213,18 @@ export class Client extends EventEmitter {
     return new Application(this, app)
   }
 
+  /** Fetch an Invite */
+  async fetch(id: string): Promise<Invite> {
```

Comment:
naming it as just `fetch` can confuse with other ones

Reply:
Changed to `fetchInvite`. I was going to do it before, but forgot.